### ISSUE TITLE
Allow the Ratchet and Clank Widescreen Patch

### ISSUE THREAD

**Issue by sandboxgamedev123** [PCSX2/pcsx2#11321]
### Description

I am proposing to allow the widescreen patch for Ratchet and Clank.

The widescreen patch for Ratchet and Clank is unavailable to use.

The file is "SCUS-97199_CE4933D0.pnach" and it is located in the "patches.zip" archive provided with the Windows version of recent nightly builds.

The following is the comment in the patch file:

"Widescreen hack by PsxFan107. Currently it appears to break rendering of textures in some areas."

I edited the patch to allow it to work in the game. I played through part of the game and I did not notice any notable texture issues as described in the comment.

URL: https://github.com/sandboxgamedev123/PCSX2-Ratchet-and-Clank-Widescreen

The URL refers to a repository I created that provides the edited patch file as well as additional information.

### Reason

The widescreen patch allows the game to operate in a 16:9 aspect ratio in-game.

### Examples

Examples would be other games for PCSX2 that have widescreen patches that are allowed.

**Comment by JordanTheToaster** [comment]
A fixed version is already being tested on the patches repo.

**Comment by Mrlinkwii** [comment]
We do not accept issues relating to Widescreen/no-interlace patches at this time.

Any issues pertaining to Widescreen/no-interlace patches please forward them to the [patches repository](https://github.com/PCSX2/pcsx2_patches).

**Comment by sandboxgamedev123** [comment]
Thank you for the information.

I did not know there was a repository for the patches.

I understand it is associated with the main PCSX2 account but maybe there should be a statement somewhere in the README.md file for PCSX2 regarding the patches repository.

**Comment by sandboxgamedev123** [comment]
> A fixed version is already being tested on the patches repo.

> We do not accept issues relating to Widescreen/no-interlace patches at this time.
> 
> Any issues pertaining to Widescreen/no-interlace patches please forward them to the [patches repository](https://github.com/PCSX2/pcsx2_patches).

Thank you for the information.

I did not know there was a repository for the patches. I understand it is associated with the main PCSX2 account but maybe there should be a statement somewhere in the README.md file for PCSX2 regarding the patches repository.

As for the patch, what was the issue before? I tested the edited version on two systems (one with an AMD CPU and another with an Intel CPU) and it appears to work well for both.

Also, I do not know why the issue was labeled as "Invalid". The patch is currently not available in the recent nightly builds.

**Comment by sandboxgamedev123** [comment]
Thank you for the information.

I did not know there is a repository for the patches. I understand it is part of the main PCSX2 account but maybe it should be mentioned somewhere in the README.md file for PCSX2.

Regarding the patch, who was having issues with it? I tested the edited version on two systems (one with an AMD CPU and one with an Intel CPU) and it appears to work well for both.

Also, why was the issue labeled as "Invalid"? The patch is currently not available in recent nightly builds.
